demo/src/main/java/com/novoda/storagepathfinder/demo/LandingActivity.kt

```diff
@@ -0,0 +1,44 @@
+package com.novoda.storagepathfinder.demo
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.support.v7.widget.LinearLayoutManager
+import com.novoda.storagepathfinder.StoragePath
+import kotlinx.android.synthetic.main.activity_landing.*
+import java.util.*
+
+class LandingActivity : AppCompatActivity() {
+
+    private var assetCloner: AssetCloner? = null
```

Comment:
The assetcloner is always gonna be nonNull after onCreate, am I right?
In that case I'd go with `lateinit var`:
`private var assetCloner: AssetCloner`
so that  later you don't have to check for null

Reply:
Yea I thought I might change this too 😄 👍